oci/private/tarball.sh.tpl

```diff
@@ -4,9 +4,20 @@ set -o pipefail -o errexit -o nounset
 readonly YQ="{{yq}}"
 readonly IMAGE_DIR="{{image_dir}}"
 readonly BLOBS_DIR="{{blobs_dir}}"
-readonly REPOTAGS='{{repotags}}'
+readonly TAGS_FILE="{{tags}}"
 readonly TARBALL_MANIFEST_PATH="{{manifest_path}}"
 
+REPOTAGS=()
+# read repotags file as array and prepend it to REPOTAGS array.
+IFS=$'\n' REPOTAGSFILE=($(cat "$TAGS_FILE"))
```

Comment:
This is currently failing because `$TAGS_FILE` doesn't exist at the expected path. I'm not sure if I am not passing it in currently to the `actions.run` that executes the shell script or maybe using the wrong path attribute.

Reply:
I changed `ctx.file.repotags.short_path` to `ctx.file.repotags.path` (the latter includes the `bazel-out/arch/bin` segments which are needed to resolve generated files)